_posts/entities/2020-07-16-func_ladder.md

```diff
@@ -0,0 +1,11 @@
+---
+title: func_ladder
+category: entity
+tags:
+- func
+- map
+- basics
+---
+----
+
+Ladder. Players will be able to freely along this brush, as if it was a ladder. If you are using a model prop for the visual representation of the ladder in the map, apply the toolsinvisibleladder material to the func_ladder brush.
```

Comment:
Actually, this entity is a default Source entity (https://developer.valvesoftware.com/wiki/Func_ladder) and does not need a momentum documentation page. It was in the FGD for parity, so this page can be deleted.